dts/arm/ti/cc1352r.dtsi

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019 Brett Witherspoon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <ti/cc13x2_cc26x2.dtsi>
+
+/ {
+	sram0: memory@20000000 {
```

Comment:
Looking at the hardware documentation, I see there are 8kb of cache memory on this device that can be optionally configured into RAM @0x11000000 by modifying ccfg.c and building it with the application. I think it needs to be defined as 'sram1'.